src/nodes.coffee

```diff
@@ -9,6 +9,12 @@ Error.stackTraceLimit = Infinity
 {RESERVED, STRICT_PROSCRIBED} = require './lexer'
 iced = require 'iced-runtime'
 
+# At this stage of development of this branch, it's easy to mess up
+# runtimes. The one in node_modules/, which is used by default, will
+# probably be incorrect.
+if not iced.const.iterator
+  throw new Error 'Wrong iced runtime!'
```

Comment:
This is now done, there's an npm module called `iced-runtime-3`.  Happy to accept PRs there.